pkg/api/v1beta3/dynakube/validation/logmodule.go

```diff
@@ -0,0 +1,48 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	errorConflictingLogModule = "The DynaKube's specification tries to enable LogModule in a namespace where another DynaKube already deploys the OneAgent, which is not supported. The conflicting DynaKube: %s"
```

Comment:
Is this error message here so specific because we know that there is a LogModule enabled?

Because I was a little confused when I first did it this way:
- apply v1beta2 and then v1beta3

and after that:

- apply v1beta3 and then v1beta2

as I got different error messages.

Reply:
Yes, that's indeed the case, this PR adds basically 2 checks:
- Block if the log agent collides with another dynakube: That's the error message you marked. 
- Block if the one agent collides with another dynakube: That's the other one. But you are right, in the other one we can also specify OneAgent, as we know this one is checked. => Will update the other message, thx

Reply:
Updated